test/interfaces/func.spec.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var Promise = require('es6-promise');
```

Comment:
why is this needed? Promises are supported for all node versions we support (anything above 4)

Reply:
IE11

Reply:
though, I don't think this test actually runs in the browser.  it should.